src/Controller/HomeController.php

```diff
@@ -22,9 +24,15 @@ public function index()
             ->setMaxResults(5)
             ->getQuery()
             ->getResult();
+        $totaloffers = $this->getDoctrine()->getRepository(Offer::class)->findAll();
```

Comment:
@MadxNC tu peux utiliser la méthode "count" ici, c'est plus adaptée

Reply:
https://ourcodeworld.com/articles/read/475/how-to-properly-count-all-the-rows-from-a-table-with-doctrine-in-symfony-4